src/components/sitepages/SiteData.vue

```diff
@@ -219,7 +227,11 @@ export default {
   watch: {
     show_user_data_only() {
       this.$store.dispatch('images/load_latest_images')
-    }
+    },
+
+    activeImageToolsTab() {
+      this.$store.commit('site_config/setActiveImageToolsTab', `${this.activeImageToolsTab}`)
```

Comment:
Looks like you have  
```
`${...}`
```
to convert the variable to a string. Is that needed/would it already be stored as a string?

Reply:
Definitely not needed - holdover from copying from a use with the subpage, which concatenated strings!